probe/probe.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 Authors of KubeArmor
+
+package probe
+
+
+import (
+	"fmt"
+	"syscall"
+    "strings"
+	"golang.org/x/mod/semver"
+
+)
+
+func PrintProbeResult() error{
+	AutoDetectKernelVersion, err := autoDetectKernelVersion()
+	if err != nil {
+		return nil
+	}
+	if AutoDetectKernelVersion == false {
+		fmt.Printf("**Host** \n")
```

Comment:
What I wanted to represent with `**Host**` is that we print Host in a **bold** or some _different_ way.